src/components/Form/Form.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+
+interface FormProps {
+  header: string
+  page: number
+  setPage: React.Dispatch<React.SetStateAction<number>>
+  inputs: Array<Inputs>
+}
+
+interface Inputs {
+  property: string
+  method: React.Dispatch<React.SetStateAction<string>>
+  placeholder: string
+}
+
+const Form: React.FC<FormProps> = ({ header, page, setPage, inputs }): JSX.Element => {
```

Comment:
I like the interfaces they are clean and clear. One things we might want to consider is restructuring react types at the top of the file. eg React.Dispatch, React.FC ...etc

Im not 100% but the import at the top would look something like this:

`import React, { FC, Dispatch, SetStateAction, FormEvent } from 'react';`

it wouldn't change much, but we wouldn't have 'React.' a bunch of times

Reply:
That's a good call, I haven't seen that before but I like it!